craft_cli/dispatcher.py

```diff
@@ -228,7 +228,11 @@ def load_command(self, app_config: Any) -> BaseCommand:
         self._loaded_command = self._command_class(app_config)
 
         # load and parse the command specific options/params
-        parser = _CustomArgumentParser(self._help_builder, prog=self._loaded_command.name)
+        parser = _CustomArgumentParser(
+            self._help_builder,
+            prog=self._loaded_command.name,
+            formatter_class=argparse.RawTextHelpFormatter,
```

Comment:
what does this do exactly?

Reply:
This disable text wrap
```
parser.add_argument(
            ...
            help="""
                1. XXX
                2. YYY
                3. ZZZ
            """
        )
```
to
`1. XXX 2. YYY 3. ZZZ`

when display.

charmcraft want to display a list of profiles with descriptions and links.

Reply:
I see. But then it sounds like it might break code that is currently relying on the default behavior of trimming spaces and wrapping words right? Maybe this should be a constructor parameter instead, so that client code can opt-in?
Also we should test this